wp-content/themes/core/assets/css/src/theme/grid/_variables.pcss

```diff
@@ -9,20 +9,49 @@
 	 * Gutters
 	 * ----------------------------------------------------------------------------- */
 
-	--grid-width-default: 1200px;
-
-	/* Gutter - Default */
-	--grid-gutter: 30px;
+	/* Gutter & Margin - Default */
+	--grid-margin: 44px;
+	--grid-gutter: 40px;
 	--grid-gutter-negative: calc(var(--grid-gutter) * -1);
 	--grid-gutter-half: calc(var(--grid-gutter) / 2);
 	--grid-gutter-half-negative: calc(var(--grid-gutter-half) * -1);
 
-	/* Gutter - Small */
-	--grid-gutter-small: 20px;
+	/* Gutter & Margin - Medium */
+	--grid-margin-medium: 44px;
+	--grid-gutter-medium: 24px;
+	--grid-gutter-medium-negative: calc(var(--grid-gutter-medium) * -1);
+	--grid-gutter-medium-half: calc(var(--grid-gutter-medium) / 2);
+	--grid-gutter-medium-half-negative: calc(var(--grid-gutter-medium-half) * -1);
+
+	/* Gutter & Margin - Small */
+	--grid-margin-small: 24px;
+	--grid-gutter-small: 24px;
+	--grid-margin-small-negative: calc(var(--grid-margin-small) * -1);
 	--grid-gutter-small-negative: calc(var(--grid-gutter-small) * -1);
 	--grid-gutter-small-half: calc(var(--grid-gutter-small) / 2);
 	--grid-gutter-small-half-negative: calc(var(--grid-gutter-small-half) * -1);
 
-	/* Widths */
-	--grid-width: calc(var(--grid-width-default) + var(--grid-gutter) * 2);
+	/* Grid Column Max Widths */
```

Comment:
I can't see ever using these, we don't actually do a traditional grid system? 😀

Reply:
Actually, I use them all the time, and have used them here. When design requests a specific width for, say an aligned image in the grid: http://p.tri.be/jQFTHd, this allows you to set the max-width to 5 columns (consistently) rather than having to recall what the pixel value for 5 columns is.

Reply:
Hah, well alrighty then. If that's the case, we should also work it into the grid as if it's only in these vars I'm sure very few people will use them.

Reply:
Can stick a pin in that for grid/layout overhaul time